From 191aff8468999abc730ffe4ee0b99fc6afd6a091 Mon Sep 17 00:00:00 2001
From: Zhang Jiejing <jiejing.zhang@freescale.com>
Date: Fri, 1 Apr 2011 10:33:26 +0800
Subject: [PATCH 873/998] ENGR00140702-2 MX5: UART: enable UART3 DMA transfer

- enable DMA transfer of UART3
- enlarge DMA receive buffer size.

Signed-off-by: Zhang Jiejing <jiejing.zhang@freescale.com>
---
 arch/arm/mach-mx5/serial.h |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/arch/arm/mach-mx5/serial.h b/arch/arm/mach-mx5/serial.h
index 9fbb0b3..6ed55da 100644
--- a/arch/arm/mach-mx5/serial.h
+++ b/arch/arm/mach-mx5/serial.h
@@ -51,8 +51,8 @@
 #define UART2_UFCR_TXTL         16
 /* UART 3 configuration */
 #define UART3_UCR4_CTSTL        16
-#define UART3_DMA_ENABLE	0
-#define UART3_DMA_RXBUFSIZE     1024
+#define UART3_DMA_ENABLE	1
+#define UART3_DMA_RXBUFSIZE     2048
 #define UART3_UFCR_RXTL         16
 #define UART3_UFCR_TXTL         16
 /* UART 4 configuration */
-- 
1.7.4.1

